cactus-maven-plugin/src/main/java/com/telenav/cactus/maven/GitPullRequestMojo.java

```diff
@@ -136,18 +136,6 @@ protected void onValidateGithubParameters(BuildLog log, MavenProject project)
         {
             fail("Base branch and target branch are the same: " + targetBranch);
         }
-        if (title != null && title.isBlank())
```

Comment:
why aren't these needed now?

Reply:
Because we synthesize a title if there is none, from a commit message (just like `gh` would if told to, but with some additional metadata).